app/src/main/res/layout/card_view.xml

```diff
@@ -0,0 +1,177 @@
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            app:layout_constraintGuide_end="8dp" />
+
+        <ImageView
+            android:id="@+id/movie_background_image_view"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:onClick="onClickMoveToDetails"
+            android:src="@drawable/movie"
+            app:layout_constraintTop_toTopOf="@id/card_view"
+            card_view:layout_constraintEnd_toEndOf="@id/card_view"
+            card_view:layout_constraintStart_toStartOf="@id/card_view" />
+
+        <View
+            android:id="@+id/gradient_view"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:background="@drawable/info_gradient"
+            app:layout_constraintBottom_toBottomOf="@id/movie_background_image_view"
+            app:layout_constraintEnd_toEndOf="@id/movie_background_image_view"
+            app:layout_constraintStart_toStartOf="@id/movie_background_image_view"
+            app:layout_constraintTop_toTopOf="@id/movie_background_image_view" />
+
+        <TextView
+            android:id="@+id/age_limit_text_view"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginTop="6dp"
+            android:alpha="0.9"
+            android:background="@drawable/age_limit_rectangle"
+            android:fontFamily="@font/roboto_black"
+            android:gravity="center"
+            android:text="@string/age_limit"
+            android:textAllCaps="true"
+            android:textColor="@color/white"
+            android:textSize="12sp"
+            app:layout_constraintStart_toStartOf="@id/guideline_start"
+            app:layout_constraintTop_toTopOf="@id/movie_background_image_view" />
+
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:alpha="0.75"
+            android:src="@drawable/ic_like"
+            app:layout_constraintBottom_toBottomOf="@id/age_limit_text_view"
+            app:layout_constraintEnd_toEndOf="@id/guideline_end"
+            app:layout_constraintTop_toTopOf="@id/age_limit_text_view" />
+
+        <TextView
+            android:id="@+id/film_genre"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="2dp"
+            android:fontFamily="@font/roboto_light"
+            android:text="@string/film_genre_drama"
+            android:textColor="@color/film_genre_color"
+            android:textSize="8sp"
+            app:layout_constraintBottom_toTopOf="@id/reviews"
+            app:layout_constraintEnd_toEndOf="@id/guideline_end"
+            app:layout_constraintStart_toStartOf="@id/guideline_start" />
+
+        <ImageView
+            android:id="@+id/first_star"
+            android:layout_width="wrap_content"
```

Comment:
use fixed height for first star
and relate other stars to first star vertically

use fixed width for all stars